src/taipy/gui/config.py

```diff
@@ -243,9 +244,20 @@ def _update(self, as_dict: t.Dict[str, t.Any]):
 
         @staticmethod
         def _configure(**properties):
+            """Configure GUI.
+
+            Parameters:
+                **properties (Dict[str, Any]): Keyword arguments that configure `Gui` behavior.<br/>
```

Comment:
"... configure `Gui` behavior." -> "... configure the behavior of the \`Gui^\` instances."